### PR TITLE
meta: Bump all semver-major dependencies

### PR DESCRIPTION
**Pull request by Swatinem**

### CODE REVIEW

File: symbolic-unreal/Cargo.toml [hunk 1]
```diff
@@ -27,7 +27,7 @@ serde = ["serde_", "chrono/serde"]
 
 [dependencies]
 anylog = "0.5.0"
-bytes = "0.4.12"
+bytes = "0.5.6"
```
Comment (jan-auer) [inline symbolic-unreal/Cargo.toml:30]: This will make it incompatible with Relay, can you check?

Comment (Swatinem) [reply symbolic-unreal/Cargo.toml:30]: yes, I think that needs updating too. Too bad I can’t use that git ref right now until we fix the breakpad submodule :-(

Comment (Swatinem) [reply symbolic-unreal/Cargo.toml:30]: relay already has both `0.4` and `0.5`, and the parameter of `symbolic-unreal` is not used anywhere.

Comment (jan-auer) [reply symbolic-unreal/Cargo.toml:30]: Thanks for verifying.

**Review by jan-auer — APPROVED** [review]

**Pull request merged**